components/RssFeedBundle/src/bundle/Form/RssFeedItemsType.php

```diff
@@ -226,7 +234,7 @@ public function getContentTypeList(): array
                 $contentTypesMap[ucfirst($contentType->getName())] = $contentType->id;
             }
             ksort($contentTypesMap);
-            if (\count($contentTypesMap)) {
```

Comment:
let `\`

Reply:
Why do we leave the '\' here?
Why don't we put it for the ksort function right on top?
How about an import?

Reply:
Seb shared an article that using `\` was quicker to resolve by the PHP

Reply:
it is and PHP Storm EAD Inspections (that you should have) is supposed to show it too ;-)